src/EnergyPlus/SizingManager.cc

```diff
@@ -3770,9 +3770,9 @@ void GetSystemSizingInput(EnergyPlusData &state)
         {
```

Comment:
A little strange to have something that is a `bool` in the IDF be an `enum` in the code. Not saying it is bad. Just a little strange. Either way, I would do this as a `bool is100PctOACooling = getYesNoValue(); SysSizInput(SysSizIndex).CoolOAOption = (is100PctOACooling) ? OAControl::AllOA : OAControl::MinOA;`.